src/main/java/exception/DukeException.java

```diff
@@ -0,0 +1,37 @@
+package exception;
```

Comment:
I've noticed that you have implemented quite a lot of subclasses that inherit from DukeException. Since the only difference between these subclasses are the error message printed, perhaps they can be combined into one subclass? (Together with a new attribute for displaying the error message)